test/fixtures/games.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+
+two:
+  name: MyString
```

Comment:
Again, unless you're using these fixtures they need to be deleted.

Reply:
Test folder has been removed in https://github.com/amykingOW/hangman/pull/2/commits/e2e800cae3f93380bc6ea02467416f2618b5c03e.